src/Sylius/Bundle/CoreBundle/Resources/config/services/commands.xml

```diff
@@ -24,22 +24,35 @@
         </service>
 
         <service id="Sylius\Bundle\CoreBundle\Command\InstallAssetsCommand">
+            <argument key="$publicDir">%sylius_core.public_dir%</argument>
```

Comment:
Why do we need `key` here?

Reply:
Cause I was using autowiring but if I remove it as you suggest above, we don't need the key anymore.